test/clj/rems/test/api/licenses.clj

```diff
@@ -26,7 +26,7 @@
                      assert-response-is-ok
                      read-body)]
         (is (coll-is-not-empty? data))
-        (is (= #{:id :start :end :licensetype :title :textcontent :localizations :attachment-id} (set (keys (first data)))))))
+        (is (:id (first data)))))
```

Comment:
Why not actually test here what the API returns?

Reply:
It is already tested by the schema validation. This test only duplicates that (and does it less thoroughly). It’s enough to check the precense of one field to know that the API is not returning a schema validation error or other junk.

Reply:
In general there is value to overlapping test coverage as long as it's not too tedious to maintain. Especially having some through the API stack (most valuable happy paths) testing which is what the API tests are mostly about.

Also why not remove all the coll-`is-not-empty?` then?

Reply:
Each test has a cost - the time it takes to run it, the time it takes to understand why the test failed, and the time it takes to fix the test when it fails for reasons unrelated to what is the focus of the test. That's why tests should be written with as much care as the production code.

Removing `coll-is-not-empty?`